docs/config/README.md

```diff
@@ -125,6 +125,36 @@ The module is only visible when the device's battery is below 10%.
 full_symbol = "🔋"
 charging_symbol = "⚡️"
 discharging_symbol = "💀"
+
+  [[battery.display]]
+  threshold = 10
+  style = "bold red"
+
+  [[battery.display]]
+  threshold = 30
+  style = "bold yellow"
+```
+
+### Battery Display
```

Comment:
Thank you for adding this section to the docs. 👍 
Much clearer now that this is here! 😄

Reply:
What are your thoughts around a sub-heading to "Battery Display" for "Options", with a similar table to option tables for modules?

It would be nice to specify that `threshold` acts as an upper bound for the `battery.display`.

Reply:
Done